### PR TITLE
feat(route): Add LA UOSC社区

### PR DESCRIPTION
**Pull request by ladeng07**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/loongarch/post
/loongarch/post/newest
/loongarch/post/top
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [x] Full text / 全文获取
  - [ ] Use cache / 使用缓存
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/loongarch/post.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                const $ = load(response.data, {
+                    scriptingEnabled: false,
+                });
+                item.author = $('#flarum-content h3:first').text();
+                item.description = $('#flarum-content .Post-body').html();
+                item.pubDate = parseDate($('meta[name="article:published_time"]').attr('content'));
+                return item;
+            })
+        )
+    );
```
Comment (TonyRL) [inline lib/routes/loongarch/post.ts:70]: Please use the API endpoint at `/api/discussions` instead.

Comment (TonyRL) [reply lib/routes/loongarch/post.ts:70]:
```ts
    const { data: response } = await got('https://bbs.loongarch.org/api/discussions', {
        searchParams: {
            include: 'user,lastPostedUser,tags,tags.parent,firstPost,firstPost',
            sort: '-createdAt',
            'page[offset]': 0,
        },
    });

    const users = response.included.filter((i) => i.type === 'users');
    const tags = response.included.filter((i) => i.type === 'tags');
    const posts = response.included.filter((i) => i.type === 'posts');

    const out = response.data.map(({ attributes, relationships }) => ({
        title: attributes.title,
        link: `https://bbs.loongarch.org/d/${attributes.slug}`,
        author: users.find((i) => i.id === relationships.user.data.id).attributes.displayName,
        description: posts.find((i) => i.id === relationships.firstPost.data.id).attributes.contentHtml,
        pubDate: parseDate(attributes.createdAt),
        updated: parseDate(attributes.lastPostedAt),
        category: relationships.tags.data.map((tag) => tags.find((i) => i.id === tag.id).attributes.name),
    }));
```
Use sort: '-commentCount' for `top`

Comment (3401797899) [reply lib/routes/loongarch/post.ts:70]: Thank you for helping me.

Comment (ladeng07) [reply lib/routes/loongarch/post.ts]: Big thanks for your assistance! I've gone ahead and refactored all the non-compliant code.

**Pull request merged**